### ISSUE TITLE
 Error #1034: Type Coercion failed - AIR 33.0.2.288

### ISSUE THREAD

**Issue by ptysbilbo** [Gamua/Adobe-Runtime-Support#132]
Hi,

## SDK version
SDK: AIR 33.0.2.288
arch: armv8

## Problem
In Flash CS6 I created a symbol and exported it to swc library as "symbols.element.GrafikWydarzenie". It extends "flash.display.MovieClip".
This symbol contains some graphics that I drew, including another of my symbols (that is exported as "symbols.opcje.Pilka" (also extends MovieClip).

In my code, when I try to create an instance:
var ikona:GrafikWydarzenie = new GrafikWydarzenie();

I get the following error:
TypeError: Error #1034: Type Coercion failed: cannot convert flash.display::MovieClip@7553e25d79 to symbols.opcje.Pilka.
	at flash.display::Sprite/constructChildren()
	at flash.display::Sprite()
	at flash.display::MovieClip()

In AIR 32, and in AIR 33 using arch: armv7 it seems to be ok, I don't get this error.

## Known Workarounds
If the symbol inside "GrafikWydarzenie" is not exported as class (so it is just a movieClip) this error doesn't happen. So for the moment I created two identical symbols: one is exported as "symbols.opcje.Pilka", and the other one is included in "GrafikWydarzenie".

**Comment by ajwfrost** [comment]
Hi - we just tried to reproduce this but it worked for us, probably we're not exactly replicating what you did. Would it be possible for you to send us your SWC file and we'll try just importing that and calling the 'new' method (or, if you send us an apk packaged with apk-debug, that would also help...)

Thanks

**Comment by ptysbilbo** [comment]
Hi, I was just about to send you an email, but when I compiled an apk few moments ago, this error didn't show anymore. I tried multiple times and it was ok. That's strange because yesterday it did happen. I didn't make any changes in the code. (Maybe it has something to do with the configuration, I use VSCode to compile, although I don't think I changed anything since yesterday).
Anyway, sorry for the trouble. If this error occurs again I will inform you.
Thanks

**Comment by ajwfrost** [comment]
Hi @ptysbilbo  - just wondering if you'd seen this issue again? If not, would you be happy to close this error report?
thanks

**Comment by ptysbilbo** [comment]
Hi, I haven't seen this issue again since my last post. However at the end, I kept the version with two identical symbols described in the 'known workarounds', because I had to commit an update to Google Play and didn't want any unexpected behaviour.

But in few other places in my swc I have symbols containing other symbols, which are exported as class and it doesn't seem to cause any problems.
So I think this issue can be closed.

Thanks
